### PR TITLE
Added NFW, CSE, and NFW_ELLIPSE_CSE profiles

### PR DESCRIPTION
**Pull request by ahuang314**
Also did some reformatting and added a comment on the epl profile.
Dropped interpol functionality from the NFW profile.

Changed the a_list and s_list in the NFW_ELLIPSE_CSE profile to be global variables instead of recreating the same list as a self variable every time a new instance of the class is created. The benefit is that we only need to convert these lists to jnp.arrays once at the very start, as opposed to each time a new instance of the class is created. This conversion is quite slow and the time loss can accumulate if lots of instances are created in e.g. slsim.

### CODE REVIEW

**Review by sibirrer — APPROVED** [review]
Thank you very much @ahuang314 !
